src/components/deviceprofileBuilder.ts

```diff
@@ -302,6 +329,31 @@ function getTranscodingProfiles(): Array<TranscodingProfile> {
     }
 
     const hlsVideoCodecs = getSupportedHLSVideoCodecs();
+
+    if (hlsVideoCodecs.length && surroundCodecs.length) {
+        // Direct streaming for passthrough codecs
+        TranscodingProfiles.push({
+            Container: 'mkv',
+            Type: DlnaProfileType.Video,
+            AudioCodec: surroundCodecs.join(','),
+            VideoCodec: hlsVideoCodecs.join(','),
+            Context: EncodingContext.Streaming,
+            Protocol: 'http'
+            //MaxAudioChannels: 6 For passthrough it's not really relevant, trying without here.
```

Comment:
No idea if this will work either, Seems silly to limit audio channels when the chromecast doesn't even care what the channels are

Reply:
Seems like it didn't hurt to leave it out